experimental/sdk/webpubsub-socketio-extension/examples/codestream/README.md

```diff
@@ -1,34 +1,34 @@
 # CodeStream, livestream your code
 
-This is a sample application to demonstrate how to use Web PubSub for Socket.IO to livestream your code to others.
+This is a sample application demonstrating how to use Web PubSub for Socket.IO to livestream your code activities to others.
 
 ## How to use
 
-1. Create an Web PubSub for Socket.IO resource
+1. Create a Web PubSub for Socket.IO resource
 2. Go to "Keys" tab and copy the connection string
 3. Run the following command with the connection string
    ```bash
    npm install
    node index.js "<connection-string>"
    ```
-4. Open `http://localhost:3000`, try to write some code in the code editor
-5. Share the link at the top of the page to others, they will see how you code in real time
+4. Open `http://localhost:3000`, try to type in some code in the code editor
+5. Share the link found at the top of the page to others, they will see your coding actitivies in realtime
 
 ## How it works
 
 ### Server side
+The express.js server does two things:
 
-The server is express.js server which only does two things:
-
-1. Serve a static web page (`public/index.html`)
-2. A REST API (`/negotiate`) which returns a url to connect to Web PubSub for Socket.IO
+1. Serves a static web page (`public/index.html`)
+2. Serves a REST API (`/negotiate`), which returns a URL to connect to Web PubSub for Socket.IO
 
 ### Client side
+Most logic is found at the client side. There are two user roles on the client side:
+- Writer
+- Viewer
 
-The most logic of this app is happening at client side. In client there're two roles:
-
-1. Streamer. Streamer is the one who writes code and broadcasts to others. It uses `io.emit("message", ...)` to send the changes from the code editor (by hooking the `editor.on('change')` event) to a room (whose ID is generated in negotiate) in Web PubSub for Socket.IO. And for performance consideration, it buffers the changes and send them in a batch every 200 milliseconds. The main implementation can be found at `startStream()` in `public/index.html`.
+1. Writer writes code in a [Monaco-powered code editor](https://microsoft.github.io/monaco-editor/) and broadcasts his/her coding actitivities to others. It uses `io.emit("message", ...)` to send the changes from the code editor to a room. For performance consideration, it buffers the changes and sends them in a batch every 200 milliseconds. The main implementation can be found at `startStream()` in `public/index.html`.
```

Comment:
do we need to mention the specific name of the editor component? It seems the detail of editor component is not quite important for users.